src/main/java/roomescape/member/Member.java

```diff
@@ -0,0 +1,41 @@
+package roomescape.member;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+public class Member {
+
+    private final static AtomicLong atomicLong = new AtomicLong(1);
```

Comment:
[많은 개발자들이 참고하는 코딩 컨벤션](https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers)을 한 번 확인해보고 프로젝트에 적용해봐도 좋을 것 같아요.
modifier (접근제한자, final, static 등...) 는 보통 특정 순서에 맞게 적어주거든요.
```suggestion
    private static final AtomicLong atomicLong = new AtomicLong(1);
```

그리고 atomicLong 의 경우, 이름을 전체 대문자로 할지, lowerCamelCase 로 할지도 컨벤션을 확인해보고 적용해보면 좋을 것 같아요!